src/components/ChatList/elements/ChannelImage.style.ts

```diff
@@ -36,16 +36,13 @@ export const channelImageStyles = StyleSheet.create({
     height: dimen.normalizeDimen(50)
   },
   containerGrey: {
-    backgroundColor: COLORS.gray9
+    backgroundColor: colors.gray1
   },
   containerGreen: {
-    backgroundColor: COLORS.holyTosca
+    backgroundColor: colors.bondi_blue
   },
   containerDarkBlue: {
-    backgroundColor: COLORS.blue
-  },
-  containerBigDarkBlue: {
-    backgroundColor: COLORS.blueSecondary
+    backgroundColor: colors.darkBlue
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been changed. Make sure these new colors (`colors.gray1`, `colors.bondi_blue`, `colors.darkBlue`) are equivalent to the old ones (`COLORS.gray9`, `COLORS.holyTosca`, `COLORS.blue`). If they are not equivalent, this could lead to unexpected changes in the app's appearance.
```diff
-    backgroundColor: COLORS.gray9
+    backgroundColor: colors.gray1

-    backgroundColor: COLORS.holyTosca
+    backgroundColor: colors.bondi_blue

-    backgroundColor: COLORS.blue
+    backgroundColor: colors.darkBlue
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->